automatic/gnucash/update.ps1

```diff
@@ -0,0 +1,29 @@
+﻿import-module au
+
+$releases = 'https://github.com/Gnucash/gnucash/releases/latest'
+
+function global:au_SearchReplace {
+  @{
+    ".\tools\chocolateyInstall.ps1" = @{
+      "(?i)(^\s*url\s*=\s*)('.*')"            = "`$1'$($Latest.URL32)'"
+      "(?i)(^\s*checksum\s*=\s*)('.*')"       = "`$1'$($Latest.Checksum32)'"
+      "(?i)(^\s*checksumType\s*=\s*)('.*')"   = "`$1'$($Latest.ChecksumType32)'"
+    }
+  }
+}
+
+function global:au_GetLatest {
+  $download_page = Invoke-WebRequest -UseBasicParsing -Uri $releases
```

Comment:
seems like the problem isn't yet gone, I just tried to run it on an appveyor build.
and it has been hanging for the last 2 minutes.